hilti/toolchain/src/config.cc.in

```diff
@@ -102,15 +101,18 @@ void Configuration::init(bool use_build_directory) {
         util::transform(hilti::util::split(prefix("${HILTI_CONFIG_RUNTIME_INCLUDE_DIRS}", "", installation_tag)),
                         [](auto s) { return hilti::rt::filesystem::path(s); });
 
-    runtime_cxx_flags_debug = flatten({prefix("${HILTI_CONFIG_RUNTIME_INCLUDE_DIRS}", "-I", installation_tag),
+    runtime_cxx_flags_debug = flatten({"-fPIC", "-std=c++17", "-g", "-fvisibility=hidden",
```

Comment:
> Can you add a comment documenting why we hide symbols by default?

Adding a comment.

> I also wonder if we should inject flags like -std=c++17 from CMake like we did previously

I moved that out of CMake to better localize it (now everything that's static is defined in the config.cc.in) and also, right, because it's really independent what CMake uses for compiling the toolchain and also potentially different then by use-case for the runtime code (JIT vs ahead-of-time).

Adding comment.